tests/integration.rs

```diff
@@ -662,6 +662,47 @@ route = "/..."
         Ok(())
     }
 
+    #[test]
+    fn spin_up_help_build_does_not_build() -> anyhow::Result<()> {
+        let env = testing_framework::TestEnvironment::<()>::boot(
+            &testing_framework::ServicesConfig::none(),
+        )?;
+
+        // We still don't see full help if there are no components.
+        let toml_text = r#"spin_version = "1"
```

Comment:
Bit surprised to see a v1 manifest here - we're generally using v2 now I think. (It's not a problem though, nothing cares whether it was loaded from v1 or v2.)

Reply:
I copy pasted this from the `up --help` works test 😅

Reply:
_chuckle_ I'll transfer my surprise there, helpfully taking it out of scope for this PR _grin_